### DISCUSSION TITLE
I'm get a 404 "File not found" on my github pages for files starting with an underscore

### DISCUSSION THREAD

**Discussion by tdegeus** [community/community#23166] [Other Feature Feedback, Questions, & Ideas]
<p>I have a GitHub pages site that uses files starting with an underscore. However, navigating to them gives me a “404 File not found” error. E.g. <a href="https://tdegeus.github.io/GooseFEM/_mesh_8h.html" rel="noopener nofollow ugc">here</a>.</p>
<p>I found a similar error <a href="https://github.community/t/multiple-pages-on-site-are-showing-up-as-404-file-not-found/11014">in this discussion</a></p>

**Comment by tdegeus** [comment, marked as answer]
<p>GitHub help clarified this for me. This problem is caused by the jekyll build system</p>
<p><a href="https://docs.github.com/en/github/working-with-github-pages/about-github-pages-and-jekyll#configuring-jekyll-in-your-github-pages-site" class="inline-onebox" rel="noopener nofollow ugc">About GitHub Pages and Jekyll - GitHub Docs</a>](<a href="https://docs.github.com/en/github/working-with-github-pages/about-github-pages-and-jekyll#configuring-jekyll-in-your-github-pages-site" class="inline-onebox" rel="noopener nofollow ugc">About GitHub Pages and Jekyll - GitHub Docs</a></p>
<p>The solution is to create a new empty file named <code>.nojekyll</code> in the root of your repository’s publishing source, which will bypass Jekyll.</p>

**Reply by pietervdvn** [reply]
This solved my issue; thx

**Reply by WaiNaat** [reply]
thank you

**Reply by maddox05** [reply]
this did not fix it

**Comment by Natheerasm** [comment]
hi
